source/index.html.md

```diff
@@ -626,14 +632,18 @@ Exactly one payment method must be used.
 
   <dt>
     series[type]
-    <span class="type"><code>recurring</code></span>
+    <span class="type">(<code>recurring</code>|<code>unscheduled</code>)</span>
```

Comment:
```suggestion
    <span class="type"><code>(recurring|unscheduled)</code></span>
```

Reply:
Is that how it's formatted elsewhere? (I thought I followed the standard of how it's done elsewhere, but I'm probably wrong 🙂)

Reply:
You're right. It's formatted with blended code and non-code in other places. Let's just keep it that way. I don't understand, however, why someone chose to not include the parentheses and the pipe inside `<code>` in the first place.

Reply:
Looking at it rendered, I acknowledge that it might have some value that the possible keywords (here "recurring" and "unscheduled") are emphasized rather than being grey like with the other regular expressions. Strictly speaking, we are (ab)using regular expression syntax, so consistency dictates 100% greyness, however, it has to be weighed against the value of emphasized values, and I tend to say the latter "wins".

Reply:
I agree completely, but I think I'll just follow whatever is apparently the default to avoid scope creep 🙂